UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs

```diff
@@ -1236,6 +1316,21 @@ private AccessorId ExportAccessor(int[] arr, bool isIndices = false)
 			return id;
 		}
 
+		private long AppendToBufferMultiplyOf4(long byteOffset, long byteLength)
```

Comment:
this can be simplified to: 		public static uint CalculateAlignment(uint currentSize, uint byteAlignment)
		{
			return (currentSize + byteAlignment - 1) / byteAlignment * byteAlignment;
		}

Reply:
for the length calculation